### PR TITLE
OPS: Update RN Device Info

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ycxdj6e6egzvgqpgjbw4vob36i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11670)

**Review by Overtorment — APPROVED** [review]

**Comment by marcosrdz** [comment]
@Overtorment  @limpbrains  detox complains it can't install the app on emulator. I can seem to figure out why. locally It works.

File: blue_modules/analytics.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ const BlueApp = require('../BlueApp');
 let userHasOptedOut = false;
 
 if (process.env.NODE_ENV !== 'development') {
+  const uniqueID = await getUniqueId();
```
Comment (limpbrains) [inline blue_modules/analytics.js:8]: You probably can't use await at the root level of the module

Comment (limpbrains) [reply blue_modules/analytics.js:8]: com.facebook.react.common.JavascriptException: ReferenceError: Property 'await' doesn't exist, js engine: hermes, stack:

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: blue_modules/analytics.js (hunk 1 above)
Comment (marcosrdz) [reply blue_modules/analytics.js]: Silly me

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ixjakedb6eqv4wutem45voomp4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11676)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**